CMakeLists.txt

```diff
@@ -517,11 +505,6 @@ endif()
 ## ARCHITECTURE
 ################################################################################
 
-if (SOLARIS OR NOT CMAKE_SIZEOF_VOID_P)
```

Comment:
mmh, is this safe? This is `OR`-combined, not `AND`-combined...

Reply:
its most probably only relevant for crosscompiling.